midp/midp.js

```diff
@@ -1261,6 +1289,9 @@ var MIDP = (function() {
     sendMediaSnapshotFinishedEvent: sendMediaSnapshotFinishedEvent,
     sendKeyPress: sendKeyPress,
     sendKeyRelease: sendKeyRelease,
+    sendDestroyMIDletEvent: sendDestroyMIDletEvent,
+    setDestroyedForRestart: setDestroyedForRestart,
```

Comment:
Nit: although this has only one caller, and that caller only calls it to set *destroyedForRestart* to `true`, that isn't obvious here in this interface definition, and a *set** function conventionally lets you set a variable to the range of possible values for its type.

So I would make this function take a boolean argument, which it assigns to *destroyedForRestart*. (Alternately, you could rename it to something like *enableDestroyedForRestart* to make it clearer that the function only sets the variable to `true`.)